coral_credits/settings.py

```diff
@@ -0,0 +1,145 @@
+"""
+Django settings for coral_credits project.
+
+Generated by 'django-admin startproject' using Django 5.0.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure--%%s+1^nkk97qur-52c0tbi4g(wxgtwuaau14etjrhvhqo0+i#"
```

Comment:
We need to remove the settings file from here I think.

@mkjpryor can talk about how he injects the configuration, typically?

Reply:
I could probably do this with values.yaml, and inject via azimuth-config: 

inject via config map: 
```
SECRET_KEY: {{ .Values.settings.secretKey | default (randAlphaNum 64) }}
```
then set in @assumptionsandg azimuth-ops role: 
```
settings:
  secretKey: "{{ azimuth_secret_key }}"
```
would probably work, unless @mkjpryor has a different suggestion?

Reply:
I could do django-flexi-settings the same as azimuth does but I'm not sure it's required yet